wiki/Installation/en.md

```diff
@@ -60,7 +60,7 @@ There are two ways to add beatmaps, via the website or osu!direct (requires [osu
         2. osu! will process the skin and automatically select it for you.
     - If the skin is compressed (using `.7z`, `.rar`, `.zip`, etc.):
         1. Extract it.
-        2. Move the skin folder into the `Skins` directory of the osu! installation.
-        3. If osu! is opened, you may need to restart it before the skin can be used.
+        2. Move the extracted skin folder into the `Skins` directory of the osu! installation path.
+        3. If osu! is running, skin will appear in the list only after you press the `Ctrl` + `Alt` + `Shift` + `S`.
```

Comment:
`after you press the` -> `after you press` (remove "the")

Reply:
```suggestion
        3. If osu! is running, skin will appear in the list only after you press `Ctrl` + `Alt` + `Shift` + `S`.
```